src/app/rundown-view/components/rundown-tooltips/tooltip-content/tooltip-content.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core'
+import { TooltipContentField } from '../../../../shared/abstractions/tooltip-content-field'
+
+@Component({
+  selector: 'sofie-tooltip-content',
+  templateUrl: './tooltip-content.component.html',
+  styleUrls: ['./tooltip-content.component.scss'],
+})
+export class TooltipContentComponent {
```

Comment:
I think you can delete this component again.
You only use it to inject content into the `PieceTooltip` component which (see my other comments) you shouldn't.

Reply:
Deleted!